src/index.js

```diff
@@ -30,6 +30,7 @@ export default class SuperImage extends React.Component {
     width       : PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
     height      : PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
     alt         : PropTypes.string,
+    role        : PropTypes.string,
```

Comment:
@tsuyoshiwada 

Aria roles which MAY be used on `img` element are only two roles, `presentation` and `none`. And only case that `alt=""`. So, role PropTypes should be better `oneOfType(['presentation', 'none'])` than `string` 😸 

Ref https://www.w3.org/TR/html-aria/#img-alt

Reply:
Hi @masuP9, Thx for reviewed !
I confirmed the specification. I'm try to fix allow only `none` or `presentation` :smiley:

Reply:
@masuP9 

In connection with PropTypes, I'd like to modify the type definitions as follows, but what do you think?

```diff
declare namespace SuperImage {
  export interface Props extends React.ImgHTMLAttributes<HTMLImageElement> {
    src: string;
    sources?: React.SourceHTMLAttributes<HTMLSourceElement>[];
+   alt?: 'none' | 'presetation';
    fit?: 'contain' | 'cover';
    fitFallback?: boolean;
    flexible?: boolean;
  }
}
```

Reply:
@tsuyoshiwada Yey! But `role?: 'none' | 'presentation'` ?

Reply:
@masuP9 Sorry, It's Typo :sob: